### ISSUE TITLE
Invalid parameter configure_etcd on class

### ISSUE THREAD

**Issue by barrymw** [cristifalcas/puppet-flannel#2]
The example has the following code, however this errors with an 'Error: Invalid parameter ... ' message:
```
    class { 'flannel':
      etcd_endpoints => "http://${::fqdn}:2379",
      etcd_prefix    => '/coreos.com/network',
      configure_etcd => true,
      network        => '172.16.0.0/16',
    }
```

Based on the lack of an etcd puppet module dependency, it doesn't appear that flannel manages etcd, as I've had to do it manually (via the etcd module).

**Comment by cristifalcas** [comment]
Correct. I removed the etcd part from flannel module, but it seams I forgot to delete this parameter also.

Thank you for spotting this.

I will update the module.
